dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -74,6 +74,31 @@ disabled.info.CWMMC0009I=CWMMC0009I: The /metrics endpoint is not available beca
 disabled.info.CWMMC0009I.explanation=The mp.metrics.prometheus.enabled MicroProfile Config property is set to false.
 disabled.info.CWMMC0009I.useraction=If the disabling of the Prometheus Registry is not intentional, remove the configuration of mp.metrics.prometheus.enabled MicroProfile Config property or explicitly set the property to true.
 
+#Embedded SmallRye Metrics or Micrometer JAR not found in the lib directory
+fileNotFound.error.CWMMC0010E=CWMMC0010E: MicroProfile Metrics was unable to initialize. The required JAR file {0} was not found in the <wlp_home>/lib directory.
+fileNotFound.error.CWMMC0010E.explanation=The JAR file was not found in the <wlp_home>/lib directory. The file may have been deleted or renamed.
```

Comment:
could we include the name of the expected jar file in this message?

Reply:
This message is used when checking for the packaged smallrye metrics JAR and packaged micrometer jar. The {0} is the param used to identify the jar. The parameter would apply to the logged message. The explanation and the user-action would have to simply refer to JAR.